files/build_templates/sonic_debian_extension.j2

```diff
@@ -143,11 +143,13 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install psutil
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install ipaddr
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install ipaddr
 
-# Install Python module for grpcio and grpcio-toole
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install "grpcio==1.39.0"
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install "grpcio-tools==1.39.0"
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install "grpcio==1.39.0"
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install "grpcio-tools==1.39.0"
+if [[ $CONFIGURED_ARCH == amd64 ]]; then
+    # Install Python module for grpcio and grpcio-tools
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install "grpcio==1.39.0"
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install "grpcio-tools==1.39.0"
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install "grpcio==1.39.0"
+    sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install "grpcio-tools==1.39.0"
```

Comment:
grpcio is supported for python3. We can also wait till #8191  is merged to address the master issue.

Reply:
I considered leaving the pip3 install in place but in the end jettisoned the whole thing. Probably not the best approach. I'm fine with waiting for PR 8191. Hoping that doesn't take too long. Don't want to not have armhf build regression coverage for too long.